topics/ui/controls/link.md

```diff
@@ -10,127 +10,266 @@
 
-</tab>
-</tabs>
+<table style="none" border="false">
+  <tr>
+    <td>
+      <format color="369650" style="bold">Correct</format>
+      <img src="link_how_to_7_correct.png" width="378" alt="Text reading 'Learn more about what is logged' where 'what is logged' is a link."/>
+    </td>
+    <td>
+      <format color="E55765" style="bold">Incorrect</format>
+      <img src="link_how_to_7_incorrect.png" width="378" alt="Link 'Learn more about what is logged'."/>
+    </td>
+  </tr>
+</table>
 
-An exception is a help topic link in [empty states](empty_state.md).
-The help icon already hints that this is an external help resource, so the arrow icon is unnecessary.
+### External link icon
 
-![](12_external_link_no_icon.png){width=164}
+Add the arrow icon for an external link. The icon shows that the user will switch to a browser and lose the current context.
+
+![Text 'Learn more about what is logged' where 'what is logged' is an external link with the external link icon.](link_how_to_8.png){width=706}
+
+<chapter title="Implementation" collapsible="true">
+
+  <tabs group="languages">
+    <tab title="Kotlin" group-key="kotlin">
+
+  ```kotlin
+  val externalLink = ActionLink("External link") {
+    BrowserUtil.browse("https://www.jetbrains.com")
+  }.apply {
+    setExternalLinkIcon()
+  }
+  ```
+  </tab>
+  <tab title="Java" group-key="java">
+
+  ```java
+  ActionLink externalLink = new ActionLink(
+      "External link",
+      event -> BrowserUtil.browse("https://www.jetbrains.com")
+  );
+  externalLink.setExternalLinkIcon();
+  ```
+  </tab>
+  </tabs>
+
+</chapter>
+
+### Help icon
+
+Use the help icon in links in [empty states](empty_state.md) to indicate that this is an external help resource. The arrow icon in this case is unnecessary.
+
+<table style="none" border="false">
+  <tr>
+    <td>
+      <format color="369650" style="bold">Correct</format>
+      <img src="link_how_to_9_correct.png" width="378" alt="Empty state in the Database tool window with an explanation text and a help source link 'Defining a database' with the help icon."/>
+    </td>
+    <td>
+      <format color="E55765" style="bold">Incorrect</format>
+      <img src="link_how_to_9_incorrect.png" width="378" alt="Empty state in the Database tool window with an explanation text and an external link 'Defining a database' with the external link icon."/>
+    </td>
+  </tr>
+</table>
 
 ### Drop-down link
 
-Drop-down links show a context menu or a popup.
-Use [`DropDownLink`](%gh-ic%/platform/platform-api/src/com/intellij/ui/components/DropDownLink.kt) to implement a drop-down link.
+Drop-down links show a context menu or a popup. To implement, use [`DropDownLink`](%gh-ic%/platform/platform-api/src/com/intellij/ui/components/DropDownLink.kt).
 
-![](13_drop_down_links.png){width=445}
+![Control 'Changes from 2 commits' where '2 commits' is a drop-down link opening a list of all commits](link_dropdown.png){width=706}
```

Comment:
The search box looks too close to the top on this illustration. I'd suggest adding 4 (or 8?) px